### PR TITLE
Feat: [OV-41716] Added the ghost secondary appearance to button

### PR DESCRIPTION
**Pull request by vicky-comeau**

### CODE REVIEW

File: .changeset/khaki-countries-occur.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@igloo-ui/button': minor
```
Comment (franckgaudin) [inline .changeset/khaki-countries-occur.md:2]: The change is major here, we replace the UI of the ghost variant which can cause regression errors.

File: packages/Button/src/Button.stories.tsx [hunk 2]
```diff
@@ -179,17 +197,37 @@ export const Sizes = () => (
...
+    </Section>
+
+    <Section>
+      <Button appearance={{ type: 'ghost', variant: 'secondary' }}>
+        Secondary
+      </Button>
+      <Button disabled appearance={{ type: 'ghost', variant: 'secondary' }}>
+        Secondary
+      </Button>
+      <Button active appearance={{ type: 'ghost', variant: 'secondary' }}>
```
Comment (franckgaudin) [inline packages/Button/src/Button.stories.tsx:221]: I don't seem to have seen an active state for the Ghost Button in Figma, to be validated with the design team

**Review by franckgaudin — COMMENTED** [review]
We must also reflect the changes in the example of the documentation.

Capture d’écran, le 2023-02-21 à 11 13 01

Still in the documentation the type ghost secondary is missing. To make this change you have to go to the file [website/data/buttonProps.json](https://github.com/gsoft-inc/ov-igloo-ui/blob/2770c73baca3f3641c0c36b1e9a288ebbfc497de/website/data/buttonProps.json#L51) and manually add
Capture d’écran, le 2023-02-21 à 11 14 28

**Comment by ofrogon** [comment]
This PR seems included in #339 no?

File: packages/Button/src/Button.stories.tsx (hunk 2 above)
Comment (vicky-comeau) [reply packages/Button/src/Button.stories.tsx:221]: I validated and there should be an active state.

**Review by ofrogon — APPROVED** [review]

**Pull request merged**